plugin/trino-iceberg/pom.xml

```diff
@@ -353,6 +368,64 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.projectnessie</groupId>
+                <artifactId>nessie-apprunner-maven-plugin</artifactId>
+                <version>${dep.nessie-apprunner.version}</version>
+                <configuration>
+                    <skip>${skipITs}</skip>
+                    <appArtifactId>org.projectnessie:nessie-quarkus:jar:runner:${dep.nessie.version}</appArtifactId>
+                    <systemProperties>
+                        <!-- disable devservices to stop Quarkus from starting Mongo via testcontainers -->
+                        <quarkus.devservices.enabled>false</quarkus.devservices.enabled>
+                        <!-- Use INMEMORY version store for testing -->
+                        <nessie.version.store.type>INMEMORY</nessie.version.store.type>
+                    </systemProperties>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.projectnessie</groupId>
+                        <artifactId>nessie-quarkus</artifactId>
+                        <version>${dep.nessie.version}</version>
+                        <classifier>runner</classifier>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>start</id>
+                        <phase>pre-integration-test</phase>
+                        <goals><goal>start</goal></goals>
+                    </execution>
+                    <execution>
+                        <id>stop</id>
+                        <phase>post-integration-test</phase>
+                        <goals><goal>stop</goal></goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.0.0-M5</version>
+                <configuration>
+                    <systemPropertyVariables>
+                        <!-- this automatically injects the random port that the nessie server is using -->
+                        <nessie.http.test-port>${quarkus.http.test-port}</nessie.http.test-port>
+                    </systemPropertyVariables>
+                    <failIfNoTests>false</failIfNoTests>
+                    <includes>
+                        <include>**/TestTrinoNessieCatalogTest.java</include>
```

Comment:
there's a rule that requires all tests to start with `Test...`

Reply:
What about the `Test` suffix here? Do we have to have it too? :)

Reply:
How about `TestTrinoNessieCatalogIntegration`? I think it would be nice to highlight the fact that this is an integration test.

The the pattern could become `<include>**/Test*Integration.java</include>`